operator-common/src/main/java/io/strimzi/operator/cluster/model/StatusDiff.java

```diff
@@ -24,7 +25,7 @@
     private final boolean isEmpty;
 
     public StatusDiff(Status current, Status desired) {
-        JsonNode source = patchMapper().valueToTree(current == null ? "{}" : current);
+        JsonNode source = patchMapper().configure(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS, true).valueToTree(current == null ? "{}" : current);
```

Comment:
Is this something we should also use in other places where we use the JsonDiff?

Reply:
I do not think that is necessary. Of course, the diff is better human-readable, if it contains multiple keys.

Reply:
I guess I don't fully understand why this line is necessary. I can see how changes to the ordering of the array `mirrorMaker2Status.getConnectors()` would affect the jsondiff (and hence why the comparator is the solution to that), but IIUC the jsondiff shouldn't care about the order of maps (see [RFC6902](https://tools.ietf.org/html/rfc6902)).

Reply:
It does not have any functional impact. It just makes debug log more readable for a human being.

Reply:
Please can we add a comment to say that.